### PR TITLE
32 modify sidebar

### PR DESCRIPTION
**Pull request by simeonc01**
Completed. 

Another bug dicovered (please check if  it's the case for you too): when loading the patients for the physician a 403 error eccors. Think this has something to do with the API connection?

### CODE REVIEW

**Comment by EmmaPeders1** [comment]
I get registered as a phycisian when i make a new user 👁️ 👁️

File: kegeland-frontend/src/routes/index.tsx [hunk 1]
```diff
@@ -25,45 +29,75 @@ export type RoutePath = {
...
+    element: (
+      <ProtectedRoutes allowedRoles={[UserRole.PATIENT]}>
+        <GamePage />
+      </ProtectedRoutes>
+    ),
   },
   {
-    path: '/patients/:patientId',
-    element: <Outlet />,
+    path: '/patients/:patientId',   // kun fysio
```
Comment (anines) [inline kegeland-frontend/src/routes/index.tsx]: I think comment should either be in our worklanguage English, or be removed

File: kegeland-frontend/src/routes/index.tsx [hunk 2]
```diff
@@ -78,10 +112,11 @@ const routes: RoutePathDefinition[] = [
     ],
   },
   {
-    title: '404',
+    title: '404',             // begge
```
Comment (anines) [inline kegeland-frontend/src/routes/index.tsx]: Same comment comment, English or remove

**Review by anines — CHANGES_REQUESTED** [review]
Looks good!

**Comment by simeonc01** [comment]
Good observation!! Comments are removed.

**Comment by kentrv** [comment]
I read all the +228,242 lines of the new code, LGTM. @simeonc01

**Review by kentrv — APPROVED** [review]

**Comment by simeonc01** [comment]
> I read all the +228,242 lines of the new code, LGTM. @simeonc01

Thanks, @kentrv !! I thought it became a bit of "spaghetti" code eventually, but since you've gone over every single line and you think it looks good, I'm satisfied :)

**Comment by anines** [comment]
@simeonc01 Will you resolve merge conflicts as well? All code looks very nice btw, good job!

**Review by anines — APPROVED** [review]

**Pull request merged**